test/connect-backbone-to-react.test.js

```diff
@@ -53,20 +64,20 @@ describe('connectBackboneToReact', function() {
   beforeEach(function() {
     sandbox = sinon.sandbox.create();
 
-    userModel = new Model({
+    userModel = new UserModel({
```

Comment:
Oh damn, that's much better. Much more work involved but definitely more robust.